types/ember/index.d.ts

```diff
@@ -3790,15 +3785,6 @@ declare module '@ember/object/proxy' {
     export default class ObjectProxy extends Ember.ObjectProxy { }
 }
 
-declare module '@ember/polyfills' {
-    import Ember from 'ember';
-    export const assign: typeof Ember.assign;
-    export const create: typeof Ember.create;
-    export const hasPropertyAccessors: typeof Ember.platform.hasPropertyAccessors;
-    export const keys: typeof Ember.keys;
```

Comment:
`keys`, `hasPropertyAccessors` and `create` have not been part of the public API for over two years (in the case of `hasPropertyAccessors` it was private in v1.13 and then removed entirely two years ago for v2.0, `create` was never public API)